### PR TITLE
Add post announcing the HCS specification and the official location

### PR DESCRIPTION
**Pull request by sbesson**
Mirrors the previous post published at the beginning of November 2020 to include a link to a representative plate using vizarr as well as a registry of all converted sample plates uploaded to EBI Embassy S3.

/cc @joshmoore @will-moore

### CODE REVIEW

**Comment by joshmoore** [comment]
For whatever reasons, it's not building for me. Not sure what's up. I'd probably also suggest bumping the Dec 1 and adding to the title just to make the transience of all this clear. Otherwise, cheers!

File: _posts/2020-11-30-zarr-hcs.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+As discussed in our
+[previous post]({{ site.baseurl }}{% post_url 2020-11-04-zarr-data %}), we
+have now extended our NGFF specification to support high-content screening.
+
+The official specification has been migrated to its own [repository](https://github.com/ome/ngff) and the current version 0.1 including
+support for multiscale images, labels and HCS is published at
+[https://ngff.openmicroscopy.org/0.1/](https://ngff.openmicroscopy.org/0.1/).
+
+All the open-source projects reading or writing the specification in various
+languages are listed in a
```
Comment (joshmoore) [inline _posts/2020-11-30-zarr-hcs.md:16]:
```suggestion
languages are listed in an
```

**Comment by will-moore** [comment]
`_data/zarr_data_2020-12-01.md` needs to be named `.json`.
And I think jekyl won't publish future blog posts, but if you also rename the post to today's date then it builds fine. 

Otherwise looks good to merge.

**Comment by will-moore** [comment]
Looks good. Builds fine. 👍

**Comment by joshmoore** [comment]
Agreed. Thanks, both. Waiting for Tuesday to release.

**Pull request merged**